contracts/erc3643/compliance/modular/modules/MaxOwnershipByCountryModule.sol

```diff
@@ -0,0 +1,283 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.24;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "../IModularCompliance.sol";
+import "../../../token/IToken.sol";
+import "./AbstractModule.sol";
+
+// This module manage the token percentage (relative to the token supply) each ONCHAINID is allowed to own
```

Comment:
I prefer to use this example to describe contract:
/**
 * @title MaxOwnershipByCountryModule
 *
 * This module manage the token percentage (relative to the token supply) each ONCHAINID is allowed to own
 */